sw/device/lib/base/testing/meson.build

```diff
@@ -12,6 +12,20 @@ sw_lib_testing_bitfield = declare_dependency(
   )
 )
 
+# global mock class
```

Comment:
nit: I don't think we really need this (not used anywhere). Maybe you can remove it when you move the `mock_abs_mmio` out of `sw/device/silicon_creator/lib/base`? You can also remove the `filegroup` for `abs_mmio.h` in `sw/device/lib/base/BUILD` as part of that change.